### PR TITLE
[WIP] Options for Microcredits

### PR DESCRIPTION
**Pull request by ggalancs**

### CODE REVIEW

File: app/admin/microcredit.rb [hunk 1]
```diff
@@ -66,6 +66,7 @@
         end
       end
       f.input :contact_phone
+      f.input :options, as: :text, :input_html => { :class => 'autogrow', :rows =>10, :cols => 10} if !resource.persisted?
```
Comment (leio10) [inline app/admin/microcredit.rb]: I'd remove this

File: app/views/microcredit/index.html.erb [hunk 2]
```diff
@@ -58,9 +58,9 @@ input.button, a.button { background-color: <%=@brand_config["color2"]%> !importa
 <% end %>
 
 <!-- % if @microcredits_mailing.length > 0 or @upcoming_microcredits_mailing.length > 0 or @finished_microcredits_mailing.length > 0 % -->
-  <!--h2><!--%= t 'microcredit.mailing.title', name: @brand_config["name"]%></h2-->
-  <!--p><!--%= t 'microcredit.mailing.help_us', name: @brand_config["name"]%--> <!--%= link_to t('microcredit.common.more_info'), (@brand_config["mailing_info_link"] || microcredits_info_mailing_path), target: "_top" %></--p>
-<!--% end %-->
+  <!--h2>< !--%= t 'microcredit.mailing.title', name: @brand_config["name"]%></h2-->
```
Comment (leio10) [inline app/views/microcredit/index.html.erb]: I'd remove all this code

File: app/admin/microcredit.rb [hunk 3]
```diff
@@ -242,3 +243,27 @@ def update
...
 end
+
+ActiveAdmin.register MicrocreditOption do
+  menu false
+  belongs_to :microcredit
+  navigation_menu :default
+
+  permit_params :microcredit_id, :location, :parent
+
+  form partial: "microcredit_option", locals: { spain: Carmen::Country.coded("ES") }
```
Comment (leio10) [inline app/admin/microcredit.rb]: I think that `locals` is not needed here

File: app/views/microcredit/index.html.erb (hunk 2 above)
Comment (ggalancs) [reply app/views/microcredit/index.html.erb]: This block is to allow microcredits of mailing type, I would keep it to allow other persons to use it only uncommenting these lines.

File: app/admin/microcredit.rb (hunk 1 above)
Comment (ggalancs) [reply app/admin/microcredit.rb]: Done!

File: app/admin/microcredit.rb (hunk 3 above)
Comment (ggalancs) [reply app/admin/microcredit.rb]: ok, removing locals part.

File: app/views/admin/microcredit_options/_microcredit_option.html.erb [hunk 4]
```diff
@@ -1,7 +1,11 @@
 <%= semantic_form_for [:admin, @microcredit_option.microcredit, @microcredit_option], builder: Formtastic::FormBuilder do |f| %>
   <%= f.inputs "Identificación" do %>
-    <%= f.input :microcredit, as: :string, input_html: { disabled: true } %>
+    <%= f.input :microcredit_id, as: :string, input_html: { disabled: true } %>
```
Comment (leio10) [inline app/views/admin/microcredit_options/_microcredit_option.html.erb]: You can change it with a hidden input

**Pull request merged**